tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -308,41 +308,52 @@ internal void ReloadEnvironmentData()
             // Merge commits have a different commit hash from the one reported by the CI.
             if (gitInfo.Commit == Commit)
             {
-                if (string.IsNullOrEmpty(AuthorName))
+                if (string.IsNullOrWhiteSpace(AuthorName) || string.IsNullOrWhiteSpace(AuthorEmail))
```

Comment:
it feels... but those issue were fixed because the test logger uses the Datadog.Trace assembly from the build (if it's available). And all changes are just improvements that I detected when checking this repo result page in CI Visibility. Felt natural to fix it while seeing the fix.

Reply:
I see... so they're only related to this change? Surely the OSVersion stuff is a general bug fix right? Not worried, just trying to understand?